python/sglang/README.md

```diff
@@ -9,4 +9,5 @@
 - `bench_serving.py`: Benchmark online serving with dynamic requests.
 - `global_config.py`: The global configs and constants.
 - `launch_server.py`: The entry point for launching the local server.
+- `launch_debug_server.py`: The entry point for launching the debug server + web app
```

Comment:
This is an experimental feature for frontend language only, so please move it under `python/sglang/lang`.